docs/build.sbt

```diff
@@ -1,8 +1,14 @@
+val ScalaVersion = "2.11.11"
 
-val PlayVersion = "2.6.6"
 val AkkaVersion = "2.5.6"
 val JUnitVersion = "4.11"
-val ScalaVersion = "2.11.11"
+val JUnitInterfaceVersion = "0.11"
+val ScalaTestVersion = "3.0.4"
+val PlayVersion = "2.6.6"
+val Log4jVersion = "2.8.2"
+val MacWireVersion = "2.2.5"
```

Comment:
Don't you need 2.3.0 for Scala 2.12 ? See the first commit of the diff between 2.2.5 and 2.3.0 versions of MacWire: https://github.com/adamw/macwire/compare/release-2.2.5...release-2.3.0